uberfire-runtime-plugins/uberfire-runtime-plugins-client/src/main/java/org/uberfire/ext/plugin/client/perspective/editor/generator/PerspectiveEditorActivity.java

```diff
@@ -16,24 +16,24 @@
 import org.uberfire.workbench.model.menu.Menus;
 import org.uberfire.workbench.model.toolbar.ToolBar;
 
-public class DefaultPerspectiveEditorActivity implements PerspectiveActivity {
+public class PerspectiveEditorActivity implements PerspectiveActivity {
 
-    private LayoutEditor editor;
-    private DefaultPerspectiveEditorScreenActivity screen;
+    private LayoutTemplate editor;
+    private PerspectiveEditorScreenActivity screen;
     private PlaceRequest place;
 
     private static final Collection<String> ROLES = Collections.emptyList();
 
     private static final Collection<String> TRAITS = Collections.emptyList();
 
-    public DefaultPerspectiveEditorActivity( final LayoutEditor editor,
-                                             DefaultPerspectiveEditorScreenActivity screen ) {
+    public PerspectiveEditorActivity(final LayoutTemplate editor,
+            PerspectiveEditorScreenActivity screen) {
         this.editor = editor;
         this.screen = screen;
     }
 
-    public void update(final LayoutEditor editor,
-                       DefaultPerspectiveEditorScreenActivity screen ){
+    public void update(final LayoutTemplate editor,
+                       PerspectiveEditorScreenActivity screen ){
```

Comment:
In your test, make sure that this method work. Update a perspective and you should see the change in apps.